packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -80,39 +81,36 @@ function decryptSpace(space) {
  */
 export function getSpaces(sparkInstance, options = {}) {
   const listOptions = Object.assign({
-    ackFilter: 'noack',
     uuidEntryFormat: true,
     personRefresh: true,
-    latestActivity: '',
     isActive: true,
     lastViewableActivityOnly: true,
     participantAckFilter: 'all',
     deferDecrypt: false,
-    // Only need 2 for One on One conversations
-    participantsLimit: 2
+    allFavorite: true,
+    participantsLimit: -1
   }, options);
   const {deferDecrypt} = options;
 
   return (dispatch) => sparkInstance.internal.conversation
     .list(listOptions)
     .then((items) => {
-      const spaces = items.map((space) => {
-        // Store users and activities
-        dispatch(storeUsers(space.participants.items));
```

Comment:
There was no need to trigger this redux store write for EVERY space if it was already decrypted. I changed it to batch them up and then do a big write at the end. The decrypted ones get written immediately because we don't want to wait for all of them to decrypt.